packages/material-ui/src/styles/transitions.js

```diff
@@ -32,7 +30,7 @@ export const duration = {
 
 export const formatMs = milliseconds => `${Math.round(milliseconds)}ms`;
 export const isString = value => typeof value === 'string';
-export const isNumber = value => !isNaN(parseFloat(value));
+export const isNumber = value => !Number.isNaN(parseFloat(value));
```

Comment:
Does is account for IE 11 support? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN

 #11743

```suggestion
export const isNumber = value => !isNaN(parseFloat(value));
```

Reply:
I doesn't. I wonder what's polyfilling it. Good reminder to check eslint-plugin-compat

Reply:
I wonder too!

Reply:
I allowed isNaN and isFinite for the time being. Airbnb uses
https://github.com/airbnb/javascript/blob/64b965efe0355c8290996ff5a675cd8fb30bf843/packages/eslint-config-airbnb-base/rules/variables.js#L19

Since Airbnb forbids it I wouldn't be surprised if enzyme somehow polyfills it.